CMSgov/mdct-mcr
# Autosave Datefields/Dropdowns

## Description


Dropdowns and datefields should now be able to autosave!

### How to test

Create a report - noting that the datefield doesn't trigger autosave but does save correctly on submission
Fill out a BSS entity to later test a dropdown autosave in sanctions
Test out a datefield and see it autosave!

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
